ElectronicObserver/Window/Control/ShipFilter/ShipFilterTranslationViewModel.cs

```diff
@@ -1,14 +1,15 @@
 ﻿using CommunityToolkit.Mvvm.ComponentModel;
 
-namespace ElectronicObserver.Window.Tools.EventLockPlanner;
+namespace ElectronicObserver.Window.Control.ShipFilter;
 
 public class ShipFilterTranslationViewModel : ObservableObject
```

Comment:
This has to inherit from `TranslationBaseViewModel` rather than `ObservableObject`, otherwise the texts don't get updated when you switch the language.